tests/filters/test_filter_fields.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import strawberry_django
+from django.db import models
+import django_filters
+
+
+class Related(models.Model):
+    pass
+
+
+class FilterModel(models.Model):
+    reverse_related = models.ForeignKey(Related, on_delete=models.CASCADE)
+    one_related = models.OneToOneField(Related, on_delete=models.CASCADE)
+    many_related = models.ManyToManyField(Related)
+    boolean = models.BooleanField()
+    char = models.CharField(max_length=50)
+    date = models.DateField()
+    date_time = models.DateTimeField()
+    decimal = models.DecimalField(decimal_places=2)
+    float = models.FloatField()
+    integer = models.IntegerField()
+    time = models.TimeField()
+    uuid = models.UUIDField()
+
+
+@pytest.mark.django_db
+def test_filter_map_types():
+    qs = FilterModel.objects.all()
+
+    @strawberry_django.create_filter
+    class Filter(django_filters.FilterSet):
+        class Meta:
+            model = FilterModel
+            exclude = ()
```

Comment:
Nice that `django_filters` can autogenerate filters.

I'm still thinking that it would be convenient to have shortcut to autogenerate filters especially at the beginning of project when you start prototyping new API. Need to think about that.

Reply:
Yeah, we can get some setting from django.config.settings. But thats the next step I would say.

Reply:
Let's do that later.